modules/spo-about.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-understanding.adoc
+
+:_content-type: CONCEPT
+[id="spo-about_{context}"]
+= About Security Profiles
+
+Security profiles can increase security at the container level in your cluster.
+
+Seccomp security profiles list the syscalls a process can make. Permissions are broader than SELinux, enabling users to restrict operations system-wide, such as `write`.
+
+SELinux security profiles provide a label-based system that restricts the access and usage of processes, applications, or files in a system. All files in an environment have labels that define permissions. SELinux profiles can define access within a given structure, such as directories.
```

Comment:
@bachradsusi Hi Petr, does the SELinux team (or the RHEL docs team in general) have a blob where we could copy a general description of what a SELinux profile is? Maybe `udica` has some docs?

Reply:
I've asked my team what would they suggest. I'd start with pointers to Getting started with SELinux -  https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/using_selinux/getting-started-with-selinux_using-selinux and Creating SELinux policies for containers - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/using_selinux/creating-selinux-policies-for-containers_using-selinux

Reply:
I also found - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/using_selinux/writing-a-custom-selinux-policy_using-selinux